api/src/test/java/org/pmiops/workbench/actionaudit/auditors/WorkspaceAuditorTest.java

```diff
@@ -117,7 +129,7 @@ public void testFiresCreateWorkspaceEvents() {
     workspaceAuditor.fireCreateAction(workspace1, WORKSPACE_1_DB_ID);
     verify(mockActionAuditService).send(eventCollectionCaptor.capture());
     Collection<ActionAuditEvent> eventsSent = eventCollectionCaptor.getValue();
-    assertThat(eventsSent).hasSize(20);
+    assertThat(eventsSent).hasSize(19);
```

Comment:
nit: Ideally we could tie this to a constant or something invariant. This is probably my fault though.